include/maths/CTimeSeriesBulkFeatures.h

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+#ifndef INCLUDED_ml_maths_CTimeSeriesBulkFeatures_h
+#define INCLUDED_ml_maths_CTimeSeriesBulkFeatures_h
+
+#include <core/CSmallVector.h>
+#include <core/CoreTypes.h>
+
+#include <maths/CBasicStatistics.h>
+#include <maths/CLinearAlgebra.h>
+#include <maths/CTypeConversions.h>
+#include <maths/MathsTypes.h>
+
+#include <boost/bind.hpp>
+#include <boost/circular_buffer.hpp>
+#include <boost/iterator/counting_iterator.hpp>
+
```

Comment:
does not seem to be used in this place

Reply:
I think this was left over from earlier versions, I'll tidy.